### ISSUE TITLE
Patience conflict on early stopping and learning rate reduce

### ISSUE THREAD

**Issue by jrrios** [zzh8829/yolov3-tf2#181]
[https://github.com/zzh8829/yolov3-tf2/blob/master/train.py#L177](url)

ReduceLROnPlateau has a default patience of 10 epochs and the patience for EarlyStopping is set to 3.

This means the learning rate will not be reduced if the loss plateaus because it will early stop before it has a chance to update the learning rate

**Comment by JimEverest** [comment]
I found this issue too, so I set the patience for EarlyStopping to 6 and patience for ReduceLROnPlateau to 3.

**Comment by jrrios** [comment]
IMO it should be configurable for the user
